features/content_lists/pin_content_list_item_on_specific_position_in_manual_list.feature

```diff
@@ -0,0 +1,410 @@
+          }
+       ],
+       "updated_at":"2050-04-29T13:59:58+00:00"
+    }
+    """
+    Then the response status code should be 201
+
+    And I am authenticated as "test.user"
+    And I add "Content-Type" header equal to "application/json"
+    Then I send a "GET" request to "/api/v2/content/lists/1/items/?sorting[position]=asc"
+    And the JSON node "total" should be equal to 4
+
+    And the JSON node "_embedded._items[0].content.title" should be equal to "Test Package4"
+    And the JSON node "_embedded._items[0].position" should be equal to 0
+    And the JSON node "_embedded._items[0].sticky" should be false
+
+    And the JSON node "_embedded._items[1].content.title" should be equal to "Test Package3"
+    And the JSON node "_embedded._items[1].position" should be equal to 1
+    And the JSON node "_embedded._items[1].sticky_position" should be null
+
+
+    Given I am authenticated as "test.user"
+    When I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/api/v2/content/push" with body:
+    """
+    {
+        "language":"en","headline":"Test Package5","version":"2","guid":"16e111d9","priority":6,"type":"text",
+        "authors":[{"name":"Tom Doe","role":"editor"}],
+        "byline":"Admin",
+        "subject":[{"name":"lawyer","code":"02002001"}]
+    }
+    """
+
+    Given I am authenticated as "test.user"
+    When I add "Content-Type" header equal to "application/json"
+    And I send a "PATCH" request to "/api/v2/content/lists/1/items/" with body:
+    """
+    {
+       "items":[
+          {
+             "content_id":5,
+             "action":"add",
+             "position":1,
+             "sticky": true
+          }
+       ],
+       "updated_at":"2050-04-29T14:05:58+00:00"
+    }
+    """
+    Then the response status code should be 201
+
+    And I am authenticated as "test.user"
+    And I add "Content-Type" header equal to "application/json"
+    Then I send a "GET" request to "/api/v2/content/lists/1/items/?sorting[position]=asc"
+    And the JSON node "total" should be equal to 5
+
+    And the JSON node "_embedded._items[0].content.title" should be equal to "Test Package4"
+    And the JSON node "_embedded._items[0].position" should be equal to 0
+    And the JSON node "_embedded._items[0].sticky" should be false
+
+    And the JSON node "_embedded._items[1].content.title" should be equal to "Test Package5"
+    And the JSON node "_embedded._items[1].position" should be equal to 1
+    And the JSON node "_embedded._items[1].sticky" should be true
+    And the JSON node "_embedded._items[1].sticky_position" should be equal to 1
+
+
+    Given I am authenticated as "test.user"
+    When I add "Content-Type" header equal to "application/json"
+    And I send a "PATCH" request to "/api/v2/content/lists/1/items/" with body:
+    """
+    {
+       "items":[
+          {
+             "content_id":1,
+             "action":"move",
+             "position":1,
```

Comment:
maybe we should use here also `stickyPosition` to be consistent with manual lists?

Reply:
this is the position you want to place the item on when it comes to the manual content list, the sticky position is a bit different because it applies only to the automatic lists and there we don't allow changing the position of other items although the position of items is saved. You can change the position of an item in an automatic content list only when you want to pin an article on the specific position so we have to keep that value somewhere thus the sticky position